tests/drivers/flash/common/testcase.yaml

```diff
@@ -45,7 +45,7 @@ tests:
     platform_allow:
       - nucleo_f103rb
       - nucleo_f207zg
-      - stm32f3_disco
+      - stm32f3_disco@B
```

Comment:
This is were this unclear to me. Is that required to define a version if a default is available ?
From your answer [here](https://github.com/zephyrproject-rtos/zephyr/pull/69568#issuecomment-1969184814) this was not required, but then why doing such change ?

Reply:
Twister needs to define the revision to use yes because it uses the identifiers from the yaml files, this is to make it clear what revision is being built for. It is users that do not, you can continue building using `west` and `cmake` without the revision and it will use the default

Reply:
Ok, so if one adds a revision to a board, all references to the board in twister should be renamed.... This isn't really great..